activities/membership-growth/membership-proposal-template.md

```diff
@@ -0,0 +1,116 @@
+
+This will:
+
+* enable higher quality public services
+* enable the pooling of resources with other public bodies to share the cost of development, maintenance and upgrading
+* build in agility for future growth paths and innovation
+* make solutions more reusable
+* gain international exposure
+
+## What becoming a member means
+
+Becoming a member means steering and becoming a co-owner of the association. This includes shaping the governance, operations and financial structure of the Foundation for Public Code.
+
+Members receive voting rights at the General Assembly, where voting rights are proportional to your member dues. For example, members vote on which codebases are stewarded by the Foundation for Public Code.
+
+Membership also includes onboarding support to help you get the most of your membership. During the onboarding we help you with your public code projects.
+
+Onboarding includes:
+
+* supporting your public code projects and codebases
+* introducing you to our network and codebase communities
+* preparing you for your first general assembly
+
+And supporting you with your codebases on:
+
+* quality of your codebases - understanding how to make them more modular and reusable
+* community for your codebases - understanding how to collaborate better and more openly with other organizations
+* product packaging of your codebases - understanding how to better market and promote your codebases
+* support of your codebases - understanding how to improve your collaboration with the market
+* [assessment of your codebases](https://about.publiccode.net/activities/codebase-stewardship/criteria-for-codebase-stewardship.html) for codebase stewardship
+
+As an association, our resources are allocated across members. Our support for each member is subject to availability.
+
+## Proposed activities
+
+[public organization] and the Foundation for Public Code will work together to identify appropriate codebases for codebase auditing, certification and stewardship activities.
+
+The Foundation for Public Code’s activities will include:
+
+* audits to help comply with the Standard for Public Code on every pull request
+* training and workshops to develop better reusable codebases as well as working with the community
+* events and regular calls to integrate a community
+* connecting relevant members of this or other communities in order to increase uptake and contributions
+* storytelling and marketing of the project based on best practices and research to support reuse
+* provide prospective reusers and contributors with a helpdesk to answer their questions on issues ranging from installations to procurement
+* help create budgets and frame costs for [public organization]’s activities outlined below
+
+[public organization] must commit time and budget to:
+
+* provide in-kind contributions of staff time to help us develop the Foundation for Public Code’s governance model, membership and financial structure
```

Comment:
This was included in the founding membership ask because:
-  we thought it would be easier to get a donation in kind than budgetary approval, based on the point in the calendar year/budgetary cycle
- we definitely need(ed?) this help/feedback

Are you deliberately including it as a permanent request?

If so, I suggest changing "develop" to sound less like we're developing it from scratch - either 'further develop' or 'improve'.

This will make this page more evergreen.